src/main/java/nextstep/optional/User.java

```diff
@@ -33,7 +35,10 @@ public static boolean ageIsInRange1(User user) {
     }
 
     public static boolean ageIsInRange2(User user) {
-        return false;
+        return Optional.ofNullable(user)
```

Comment:
아래와 같이 변경해도 예외는 발생하지 않고, 의도대로 잘 동작합니다.
왜 null 체크를 하지 않아도 괜찮을까요? :)
`Optional` 에서 제공하는 `filter`, `map` 메소드 구현을 확인해보시면 도움이 될 것 같습니다!

```suggestion
return Optional.ofNullable(user)
    .map(User::getAge)
    .filter(age -> age >= 30 && age <= 45)
    .ifPresent();
```

Reply:
`isPresent` 가 널체크를 대신 해주어서 괜찮은 것 같습니다. 🙂